cloudfront-s3-cdk-go/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+    "title": "CloudFront Distribution in front of S3 Static Website",
+    "description": "Create an Amazon S3 Static Website with a CloudFront Distribution in front of it.",
+    "language": "Go",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This pattern shows how to Create an Amazon S3 Static Website with a CloudFront Distribution in front of it. Implemented in AWS CDK using the Go programming language.",
+        "This pattern deploys one CloudFront Distribution and an S3 Static Website."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cloudformation-s3-cdk-go",
+        "templateURL": "serverless-patterns/cloudformation-s3-cdk-go",
```

Comment:
Please remove the projectFolder value from the templateFile. Please refer to some of the other Go examples that we have under Serverless patterns. 
```
        "template": {
            "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cloudformation-s3-cdk-go",
            "templateURL": "serverless-patterns/cloudformation-s3-cdk-go",
            "projectFolder": "cloudfront-s3-cdk-go",
            "templateFile": "cloudformation-s3-cdk-go.go"
        }
        
        ```

Reply:
Hi @pputhran I have removed the projectFolder value and fixed a few cloudformation/cloudfront typos.  

But could you point me toward one of the relevant go examples?  None of the go cdk examples I see have an example-pattern.json present.